pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -271,7 +271,7 @@ let
         make modules_install $makeFlags "''${makeFlagsArray[@]}" \
           $installFlags "''${installFlagsArray[@]}"
         unlink $out/lib/modules/${modDirVersion}/build
-        unlink $out/lib/modules/${modDirVersion}/source
```

Comment:
Maybe just `rm -f` here?

Reply:
That does look cleaner, I'll amend the PR in a few minutes.